pkg/kubectl/cmd/config/set_authinfo.go

```diff
@@ -56,7 +56,7 @@ const (
 )
 
 var (
-	create_authinfo_long = fmt.Sprintf(templates.LongDesc(`
+	createAuthinfoLong = fmt.Sprintf(templates.LongDesc(`
```

Comment:
> createAuthinfoLong

If you change the file name, you'd better change this and below to make all the variables consistent with the file name.
Same for all the files with name changed.